common/config/settings.py

```
@@ -1308,6 +1308,12 @@ def normalize_log_level(cls, v):
         description="Enable JSON mode for LLM reviewer (ensures valid JSON responses)"
     )
 
+    llm_review_max_diff_chars: int = Field(
+        default=80000,
+        alias="LLM_REVIEW_MAX_DIFF_CHARS",
+        description="Maximum characters for annotated diff in LLM reviewer prompt. Diffs exceeding this limit will be truncated (preserving + lines over context lines). Default 80k chars (~20k tokens) leaves room for system prompt and response."
+    )
+
```

IMPACT: Adding a new config field without a corresponding version bump in settings/schema breaks backward compatibility for configuration management and deployment systems expecting schema stability. This can cause silent failures or misconfigurations in environments relying on schema validation. SUGGESTED FIX: Introduce a schema version field (e.g., SETTINGS_SCHEMA_VERSION) and increment it, or document this as a breaking change requiring coordinated deployment.